frontend/src/lib/components/ResizableTable/TableConfig.scss

```diff
@@ -24,7 +23,7 @@
         }
 
         &.selected {
-            background-color: var(--primary-hover);
+            background-color: var(--bg-light-blue);
```

Comment:
This should be `--primary-bg-hover`, like [said here](https://github.com/PostHog/posthog/pull/10293#discussion_r897004326). 

That color is defined in a roundabout way [here](https://github.com/PostHog/posthog/blob/fd11b357c09b76a50fd0c368cce5874966b57520/frontend/src/styles/global.scss#L19). It's actually a strong blue with an alpha channel, not a light blue :D.